index.js

```diff
@@ -263,35 +222,35 @@ module.exports = {
             "grid-column-start",
             "grid-column-end",
             "grid-row-start",
-            "grid-row-end"
-          ]
+            "grid-row-end",
+          ],
         },
         {
-          "emptyLineBefore": "always",
-          "properties": [
+          emptyLineBefore: "always",
+          properties: [
             "box-sizing",
             "visibility",
             "clear",
             "overflow",
             "overflow-x",
             "overflow-y",
             "clip",
-            "zoom"
-          ]
+            "zoom",
+          ],
         },
         {
-          "emptyLineBefore": "always",
-          "properties": [
+          emptyLineBefore: "always",
+          properties: [
             "float",
             "clear",
             "vertical-align",
             "align-self",
-            "justify-self"
-          ]
+            "justify-self",
+          ],
```

Comment:
I just wonder, why do we add `,` to last items? Is it your practice or did you run a formatter?